CHANGELOG.md

```diff
@@ -39,10 +41,16 @@ _ `_optional` subpackage for managing optional dependencies
 - Sampling methods in `qNIPV` and `BotorchRecommender` are now specified via 
   `DiscreteSamplingMethod` enum
 - `Interval` class now supports degenerate intervals containing only one element
+- Context information required by `Surrogate` models is now cleanly encapsulated into
+  a `context` object passed to `Surrogate._fit`
+- Fallback models created by `catch_constant_targets` are stored outside of surrogate,
```

Comment:
I agree that most of the other entries are shorter / do not mention reasons, but I think that was mostly because they were typically obvious. In general, I wouldn't have anything against having more informative entries if otherwise the motivation for the change would be unclear. But let's hear @Scienfitz's opinion

Reply:
I also don't really see the need here for what comes after `which`, I'd simply remove it and add a `now` between `are stored`

Reply:
removed

Reply:
However, having more informative Changelog entries in general might be worth having, but I'd suggest that we (i) discuss first if we all agree with what is "more informative" and (ii) change that after a release and not in between two.